keys/sources.go

```diff
@@ -0,0 +1,19 @@
+package keys
+
+var (
+	// StagingTokenPublicKeySource represents the url to the JWT public key for the staging environment
+	StagingTokenPublicKeySource = HTTPSource("https://api-gateway-staging.platformserviceaccount.com/token/public")
```

Comment:
I'd rather these be parameterised somewhere, to keep the core usable by everyone, not just Lush.

Perhaps these can be shipped as an optional default in some way, or just removed altogether

Reply:
That's interesting, I thought this package was mostly public due to the ease of packaging. If not in code, where would you suggest we provide this information? I specifically thought of having them in a dependent package so that it would be easy to manage.

Reply:
After discussion with teapot, we agreed that we will provide the URL source through the platform environment.